méli-mélo/2023-10-clipboard/index.html

```diff
@@ -0,0 +1,386 @@
+<br>
+<pre class="prettyprint"><code>&lt;div class=&quot;wb-clipboard&quot; data-wb-clipboard=&apos;{&quot;noTextStyle&quot;: true, &quot;addBorder&quot;: false, &quot;copyBtnStyle&quot;: &quot;btn btn-primary wb-clipboard-btn&quot;, &quot;copiedBtnStyle&quot;: &quot;btn btn-info wb-clipboard-btn&quot;}&apos;&gt;
+    &lt;p&gt;In maximus orci quis ex maximus, sed rhoncus arcu ullamcorper. Mauris tincidunt sodales magna vitae luctus. Cras cursus dapibus arcu sed commodo. In enim diam, molestie in mattis nec, hendrerit et sapien. Curabitur nec lorem dapibus, egestas diam vitae, sollicitudin erat.&lt;/p&gt;
+&lt;/div&gt;</code></pre>
+<!-- Copy to Clipboard Example 5 -->
+<h3>Custom copy area start/end text</h3>
+<p>Change the hidden text surrounding the copyable area. Allows different values for before and after the content is copied.</p>
+<div class="wb-clipboard"  data-wb-clipboard="{&quot;clipStartText&quot;: &quot;clipped content starts here&quot;, &quot;clipEndText&quot;: &quot;clipped content ends here&quot;, &quot;copiedStartText&quot;: &quot;copied content starts here&quot;, &quot;copiedEndText&quot;: &quot;copied content ends here&quot;}"><p>Quisque rhoncus tortor nibh, non cursus purus vulputate a. Ut ac nisi enim. Donec eu libero dui. Aenean in augue velit. Vestibulum at tellus fermentum lacus viverra mollis. Vestibulum ut fringilla lacus. Nullam cursus, elit quis vehicula maximus, tortor sem bibendum sem, ac rutrum est libero et nisl.</p></div>
+<br>
+<pre class="prettyprint"><code>&lt;div class=&quot;wb-clipboard&quot; data-wb-clipboard=&apos;{&quot;clipStartText&quot;: &quot;clipped content starts here&quot;, &quot;clipEndText&quot;: &quot;clipped content ends here&quot;, &quot;copiedStartText&quot;: &quot;copied content starts here&quot;, &quot;copiedEndText&quot;: &quot;copied content ends here&quot;}&apos;&gt;
+    &lt;p&gt;Quisque rhoncus tortor nibh, non cursus purus vulputate a. Ut ac nisi enim. Donec eu libero dui. Aenean in augue velit. Vestibulum at tellus fermentum lacus viverra mollis. Vestibulum ut fringilla lacus. Nullam cursus, elit quis vehicula maximus, tortor sem bibendum sem, ac rutrum est libero et nisl.&lt;/p&gt;
+&lt;/div&gt;</code></pre>
+<!-- Copy to Clipboard Example 6 -->
+<h3 id="noButton">Deactivate clipboard button</h3>
+<div class="wb-clipboard" data-wb-clipboard="{&quot;noButton&quot;: true}">
+    <p>Sed fermentum ante ipsum, nec pulvinar dui maximus at. Sed euismod nisl urna, in pretium elit ornare vel. Nunc pharetra commodo enim, et molestie nulla volutpat eu. Duis pulvinar vulputate tortor, quis venenatis libero pharetra nec. Nulla convallis eu turpis quis dignissim. In fermentum porta fringilla. Nulla vel ante a erat efficitur placerat. Curabitur gravida finibus erat a rutrum.</p>
+</div>
```

Comment:
**Suggestion:** As it stands currently, I do not think that the `noButton` should be kept as-is since it doesn't have any visual nor semantic indicator that the section is a "Copy to clipboard" section or that it is interactive at all, which to me raises a usability concern, if not accessibility. I recommend to remove this option until further investigation.

Reply:
Will review in a future discussion